libs/application/templates/driving-license/src/forms/review.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  buildForm,
+  buildIntroductionField,
+  buildMultiField,
+  buildSection,
```

Comment:
All of the forms are importing a lot of stuff they are not using

Reply:
true... will fix it in the next slice, since the design isn't ready.